src/main/java/cloud/fogbow/rcs/core/intercomponent/xmpp/XmppComponentManager.java

```diff
@@ -78,6 +79,7 @@ private XmppComponentManager(Builder builder) {
         
         // instantiate get handlers here
         addGetHandler(new RemoteGetServiceRequestHandler());
+        addGetHandler(new RemoteGetServicesRequestHandler());
```

Comment:
A tip: to avoid confusion, maybe you should change the name of this class to **RemoteGetAllServicesRequestHandler**.